### DISCUSSION TITLE
any plan on windows version?

### DISCUSSION THREAD

**Discussion by blackeyed7** [Chia-Network/bladebit#17] [General]
I am really looking forward to testing this on my windows server
would you like to make a windows version? Many thanks

**Comment by harold-b** [comment]
The intention was there initially, but I probably won't. I don't have a machine to develop in and I don't plan on continuing paying for cloud systems to develop this, especially with me not being able to use it.

**Reply by wsobrinho** [reply]
Hello harold,

To develop would you need an environment with 512 RAM?
Or would it be possible to run tests with low memory using smallest k plot? 

Another question,,, On a machine with 512 Ram what does it take to create a development environment? Vscode, Visual Studio ? Docker? The Windows subsystem for Linux looks very interesting as it compiles directly into a ubuntu

**Reply by harold-b** [reply]
It's not possible to use smaller k's. It is explicitly set up to run with k32 only, so yeah, 416 GiB of RAM is required to run it (so yes, more to develop it).

As for a dev env, the project files for VSCode and Visual Studio are generated from an app called psync (details in the README) which I've not yet open sourced. The current Visual Studio project files have not been updated as I've not touched the windows version in quite a bit.

Using WSL is fine for developing on linux, I use it myself. VScode c_cpp_properties.json file is up to date for linux.
